### DISCUSSION TITLE
Livestream could be possible now we have the increased speed!

### DISCUSSION THREAD

**Discussion by loawizard** [nagadomi/nunif#194] [General]
Thank you so much for increasing the speed I get over 30fps on any v2 now which means I can basically livestream anyv2 mkv files to my 3D projector. It's the dream!

I was wondering if it would be possible to to have a lifestream from an url getting rendered as it's streaming to an .MKV

Now the Olympics are on , that would be great thing to watch in 3D instantly on my projector using an .mkv  file and novaplayer

**Comment by nagadomi** [comment]
In theory, URLs that can be loaded by ffmpeg should be able to be processed.
But I'm not familiar with live streaming protocols.
I think recent livestream is HLS (m3u8), but I don't know if it has a single URL.
If it is RTSP, it can probably be processed.
Anyway, I need to do some research.

**Reply by loawizard** [reply]
It would be awesome feature.
